docs/conf.py

```diff
@@ -196,7 +196,11 @@
 .. raw:: html
 
     <div>
```

Comment:
Oh, one more thing; if we add
```suggestion
    <div class="admonition info">
```

Then these boxes are highlighted slightly more separately:

![image](https://user-images.githubusercontent.com/1203825/80945233-b71d1e00-8e2e-11ea-98b8-211dfa28d528.png)

Reply:
I added this; can't decide if it's needed with the other change though?

Reply:
I think it helps especially at the end at the end to avoid the text running into the action buttons.